config/routes.rb

```diff
@@ -6,6 +6,9 @@
    root to: "users#index"
 
    resources :users, only: [:index, :show] do
-      resources :posts, only: [:index, :show]
+      resources :posts, only: [:index, :show , :new, :create] do
+        resources :comments, only: [:create]
```

Comment:
- [ ] Nice work for the routes here, but please add a route for the comment `new` method as you added for `create`.